File: program9/main.go
package main

import (
	"fmt"
)

func main() {
	var l1 []int

	var n int
	fmt.Print("How many elements do you want? ")
	fmt.Scan(&n)

	for i := 0; i < n; i++ {
		var x int
		fmt.Printf("Enter element %d of the list: ", i+1)
		fmt.Scan(&x)
		l1 = append(l1, x)
	}

	maxe := l1[0]
	imax := 0

	for i := 1; i < len(l1); i++ {
		if l1[i] > maxe {
			maxe = l1[i]
			imax = i
		}
	}

	if imax > len(l1)/2 {
		fmt.Println("Maximum element lies in the second half")
	} else if imax < len(l1)/2 {
		fmt.Println("Maximum element lies in the first half")
	}
}

File: program9/main.py
l1=[]
n=int(input("How many elements do you want? "))
for i in range(n):
    x=int(input(f"Enter element {i+1} of the list: "))
    l1.append(x)
maxe=max(l1)
imax=l1.index(maxe)
if imax > len(l1)//2:
    print("Maximum element lies in second half")
elif imax < len(l1)//2:
    print("Maximum element lies in first half")
